### PR TITLE
Feature/add price tags

### PR DESCRIPTION
**Pull request by lucaspacheco-acct**
#### What problem is this solving?

Add the field `priceTags` on type `Items` to make possible to see `priceTags` on `ItemsContext`, so we can differ products with same id

#### How should this be manually tested?

[Workspace](https://pac--carrefourbrfood.myvtex.com/chocolate-ao-leite-bis-126g-9284230/p)
Access the workspace above and add four units of this product on cart, if you still didn't regionalize you won't be able to add the product to the cart, you can use this postal code for regionalize: 04814-620. Then open the minicart and see if the free product is in the cart. Then you can run the following query: https://pac--carrefourbrfood.myvtex.com/_v/private/vtex.checkout-graphql@0.41.1/graphiql/v1?operationName=orderForm&query=query%20orderForm%20%7B%0A%20%20orderForm%20%7B%0A%20%20%20%20items%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20priceTags%20%7B%0A%20%20%20%20%20%20%20%20identifier%0A%20%20%20%20%20%20%20%20isPercentual%0A%20%20%20%20%20%20%20%20rawValue%0A%20%20%20%20%20%20%20%20value%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20ratesAndBenefitsIdentifier%20%7B%0A%20%20%20%20%20%20%20%20%20%20description%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20featured%0A%20%20%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20%20%20matchedParameters%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A

The splitted product:
image

The query response with items priceTags:
image

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```
Comment (lucasecdb) [inline CHANGELOG.md:9]:
```suggestion
```

File: graphql/types/Item.graphql [hunk 2]
```diff
@@ -43,6 +43,7 @@ type Item {
   skuSpecifications: [SKUSpecification!]!
   uniqueId: String!
   unitMultiplier: Float
+  priceTags: [PriceTags]
```
Comment (lucasecdb) [inline graphql/types/Item.graphql:46]:
```suggestion
  priceTags: [PriceTags!]!
```

Comment (lucaspacheco-acct) [reply graphql/types/Item.graphql:46]: @lucasecdb setting `priceTags` as required can't be dangerous for websites that uses the `orderForm` query in production?

Comment (lucasecdb) [reply graphql/types/Item.graphql:46]: doesn't it always return an empty array? and this won't affect any stores unless we update the orderForm query in checkout-resources

Comment (lucaspacheco-acct) [reply graphql/types/Item.graphql:46]: @lucasecdb actually we have another PR that adds `priceTags` on `ItemFragment` in checkout-resources.
https://github.com/vtex-apps/checkout-resources/pull/58

Comment (lucasecdb) [reply graphql/types/Item.graphql:46]: oh, can you name the type in singular? like `PriceTag`

Comment (lucaspacheco-acct) [reply graphql/types/Item.graphql]: @lucasecdb all suggestions was applied :)

**Review by lucasecdb — APPROVED** [review]

**Comment by lucasecdb** [comment]
@lucaspacheco-acct can you rebase against master and remove the merge commit 85dd4a2?

**Comment by lucaspacheco-acct** [comment]
@lucasecdb done

**Comment by lucasecdb** [comment]
@lucaspacheco-acct the merge commits are still there, I tried to do it myself but I don't have write permissions on the fork repository. You need to do a rebase on your branch commits and drop the ones that weren't yours:

You can run:

```sh
git rebase -i ab358f55bcd37ddb39774099d369d5d75993b53d~1
```

It will then open a file in your configured editor, you should edit it to drop the commits that came with the merge, so it will look like this:

```gitrebase
pick ab358f5 added priceTags field on Item type
pick 3111339 changelog update
pick 9da1bb3 change PriceTag name
pick d5bdb56 Update CHANGELOG.md
pick 0f53322 set priceTags as required
d 048d74a Add orderFormId argument to queries and mutations
d 9a8553f Release v0.42.0
d f61323f Adding shippingSLA query
d 91b9741 Addressing @lucasecdb suggestions
d e2ed3cf Using the same util function now mocking the OF
d 85b33ed Release v0.43.0-beta.0
d 59e8d60 Linting and improving types
d 912d6f1 Release v0.43.0
d 539c25d Add pickupOptions to shipping type and selectPickupOption mutation
d 02edc0f Optional argument `itemId` on `selectPickupOption`
d 97b4e12 Update changelog
d abdc57d Update tests
d 5111444 Use ternary to compare incoming itemId when selecting delivery option
d 0a33601 Update node/constants/index.ts
d dedca29 Update PICKUP_IN_POINT variable name
d 236f4ba Release v0.44.0
d 4c6dea9 Fix error when shippingData in orderForm is null
d 38f058e Update PR template
d fc6dd08 Fix shippingData typings
d e559138 Release v0.44.1
d f03cfad Revert "Merge branch 'feature/add-price-tags' of https://github.com/ACCT-global/checkout-graphql into feature/add-price-tags"
```

Save this and exit, and git will drop the other commits. The only thing left to do is to push the commits with `git push -f` (the force option is because we've _rewritten_ the history in order to drop these commits, so if you try to do a normal git push, git would reject it because it would see your branch as like it's not up-to-date with the remote and will suggest you to do `git pull` before pushing, but you should **not** do that because you will end up with duplicated commits)

**Review by lucasecdb — APPROVED** [review]

**Comment by lucaspacheco-acct** [comment]
@lucasecdb Thanks, I believe that's all right with commits now :D

**Comment by lucasecdb** [comment]
@lucaspacheco-acct yeah! 😄 thanks for the pull-request :-)

**Review by alvimm — APPROVED** [review]

**Pull request merged**